apps/web/package.json

```diff
@@ -75,7 +75,7 @@
     "@gc-digital-talent/jest-helpers": "workspace:*",
     "@gc-digital-talent/jest-presets": "workspace:*",
     "@gc-digital-talent/storybook-helpers": "workspace:*",
-    "@hydrogen-css/hydrogen": "2.0.5",
+    "@hydrogen-css/hydrogen": "github:hydrogen-css/hydrogen#2.1.0",
```

Comment:
Is dependabot going to be able to manage this?

Reply:
@petertgiles I mentioned this in the "note" block above, but it won't be staying this way before we merge. I'm just using this to ensure I can push up changes to the release branch and pull them down here really rapidly. When we're happy with the outcomes, I'll properly publish the release and update the values here.

Reply:
Ah, sorry, I skimmed and missed it.  :grimacing: Thanks!